src/style.css

```diff
@@ -90,3 +90,8 @@ body {
     width: 100%;
   }
 }
+
+.my-like-button {
+  width: 50px;
+  height: 50px;
+}
```

Comment:
[OPTIONAL] I think it would be better to add some margin-bottom to get a better design. 😍 

![9](https://user-images.githubusercontent.com/85481456/233090100-b0c36cab-f18e-452a-ae1d-b53133dba7d7.png)